thrust/thrust/tuple.h

```diff
@@ -98,18 +99,22 @@ _CCCL_HOST_DEVICE inline bool operator>(const null_type&, const null_type&)
  *  \see tuple
  */
 template <size_t N, class T>
-using tuple_element = ::cuda::std::tuple_element<N, T>;
+using tuple_element = _CUDA_VSTD::tuple_element<N, T>;
 
 /*! This metafunction returns the number of elements
  *  of a \p tuple type of interest.
  *
  *  \tparam T A \c tuple type of interest.
  *
- *  \see pair
+ *  \see tuple
  *  \see tuple
```

Comment:
Question: unintended replace? Docs linked to `pair` and `tuple` before and now link to `tuple` twice.